pkg/storage/db.go

```diff
@@ -739,7 +753,14 @@ func (db *DB) runQuery(ctx context.Context, query string, args ...interface{}) (
 		var isJoin bool      // no need for sql.NullBool as the DB column is guaranteed a value
 		var timestamp time.Time
 
-		err = rows.Scan(&row.ARN, &ipAddress, &hostname, &isPublic, &isJoin, &timestamp, &row.AccountID, &row.Region, &row.ResourceType, &metaBytes)
+		if version < DualWritesSchemaVersion {
+			err = rows.Scan(&row.ARN, &ipAddress, &hostname, &isPublic, &isJoin, &timestamp, &row.AccountID, &row.Region, &row.ResourceType, &metaBytes)
+		} else {
+			err = rows.Scan(&ipAddress, &hostname, &row.ARN, &metaBytes, &row.Region, &row.ResourceType, &row.AccountID)
+			ip, _, _ := net.ParseCIDR(ipAddress)
+			isPublic = isPublicIP(ip)
```

Comment:
Private IPs are in different table. The returned by this query is always public.

Reply:
True. I was preparing for when we implement resourceByPrivateIPQuery, I assume what it will return will be mapped to the same variable name as the public ip query ("ipAddress"). 
But I can change it to be static (isPublic = true) on this PR if that keeps things cleaner.